stemcell_builder/stages/static_libraries_config/assets/xenial_static_libraries_list.txt

```diff
@@ -75,3 +75,5 @@
 /usr/lib/x86_64-linux-gnu/liby.a
 /usr/lib/x86_64-linux-gnu/libz.a
 /usr/src/linux-headers-4.15/tools/objtool/libsubcmd.a
+/usr/local/ssl/lib/libcrypto.a
```

Comment:
These two libraries seemed to be added to the global list of `xenial_static_libraries`, but this applies to all stemcells that are being built. Can you explain the reason for this change?